### ISSUE TITLE
Improvement: Default Collection for All Users on Top

### ISSUE THREAD

**Issue by ajs-codes** [ajs-codes/cinelog#123]
when an user is signed up in after() block of the next response api we need to create an collection for Continue Watching based on 
- series type 
- sorted by date last watched 
- group by none 
- show in dashboard = true
- show in library false
- order should be 1

so for every user there will be an default collection if they don't want they can modify, reorder or delete it.
